python/ray/services.py

```diff
@@ -19,6 +20,11 @@
 import ray.ray_constants as ray_constants
 import psutil
 
+libc = None
+prctl = None
+PR_SET_PDEATHSIG = 1
```

Comment:
No need to make this a global constant if it's only used in one specific context

Reply:
I think it's better this way? It looks like a magic number otherwise.

Reply:
If it's only used in one place it's better to just have a local variable instead - that way you don't have to track down where the constant is defined.